test/unit/factorization/mc/test_cholesky.cpp

```diff
@@ -145,10 +145,6 @@ void testDistCholesky(CholeskyDistributedTest<TypeParam>& test) {
   }
 }
 
-TYPED_TEST(CholeskyDistributedTest, DistCholesky_Yielding) {
```

Comment:
@teonnik am I wrong saying it is duplicated? did am I missing something here?

Reply:
Yes, you are right, one of them should be removed!